outboundSender_test.go

```diff
@@ -918,3 +919,70 @@ func TestOverflow(t *testing.T) {
 
 	assert.NotNil(output.String())
 }
+
+func TestDispatcherEmptyQueue(t *testing.T) {
+	fmt.Printf("\n\nTestingDispatcherEmptyQueue:\n\n")
```

Comment:
```suggestion
```
Adding this is unnecessary as we can run go test  -v to get which test it was apart of